src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -850,13 +851,33 @@ In the Job xml schema it is available as `preciousResult`:
 
 ==== Job Result Map
 
-Job has key-value map associated to it, which is called *Job Result Map*. Each task can contribute to this *Job Result Map*. Consider the following groovy example:
+Job has key-value map associated to it, which is called *Job Result Map*.
+Each task can contribute to this *Job Result Map*. It is write-only map,
```

Comment:
It is a write-only map, in such a way that a task cannot see what other tasks already wrote to the result map.